### PR TITLE
Delivering orders.

### PR DESCRIPTION
**Pull request by konrad04-1989**
Hello,

see the new features on Loom! :)
[https://www.loom.com/share/8e8a73a21db64dd8982cb638c2db07d0](https://www.loom.com/share/8e8a73a21db64dd8982cb638c2db07d0)

### CODE REVIEW

File: app/controllers/admin/orders_controller.rb [hunk 1]
```diff
@@ -0,0 +1,16 @@
...
+
+module Admin
+  class OrdersController < AdminController
+    def index
+      @orders = Order.all.includes([:employee]).order(:status)
+    end
+
+    def update
+      @order = Order.find(params[:id])
+      @order.status = 1
```
Comment (ciejj) [inline app/controllers/admin/orders_controller.rb]: try using helper method offered by enum here - something like
```suggestion
      @order.status_delivered!
```

File: app/models/order.rb [hunk 2]
```diff
@@ -2,6 +2,7 @@
 
 class Order < ApplicationRecord
   serialize :snapshot
+  enum status: { not_delivered: 0, delivered: 1 }, _prefix: :status, _default: 'not_delivered'
```
Comment (ciejj) [inline app/models/order.rb:5]: I am not sure about this prefix - I know that many articles are saying that it is making reading easier, but I personally find it more confusing, for me in this case order.delivered is self explanatory - but its up to you if you want ot keep it, or leave it ;)

**Review by ciejj — APPROVED** [review]
Well done - left some personal thoughts about enums ;)

also - try make the messages of commits a bit more descriptive for example:- instead of 'routes updated' try 'add orders index and update route' - check out the routes file history, and tell when was some particular routes added ;)

File: app/controllers/admin/orders_controller.rb (hunk 1 above)
Comment (konrad04-1989) [reply app/controllers/admin/orders_controller.rb]: I have revised this method. Thank you for your advice. 

As for the commit descriptions - I promise to improve ;) I'll remind you at the next PR to see if anything else can be improved.

File: app/models/order.rb (hunk 2 above)
Comment (konrad04-1989) [reply app/models/order.rb:5]: OK. Thanks for the tip. I'll leave it as is for now. I wanted to test how it works. At the next opportunity I won't add it unnecessarily.

**Pull request merged**